src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift

```diff
@@ -36,6 +36,20 @@ class HealthCertificateService {
 				registerHealthCertificate(base45: HealthCertificate.lastBase45Mock)
 			}
 			
+			if LaunchArguments.healthCertificate.invalidCertificateCheck.boolValue {
+				let dcc = DigitalCovidCertificateFake.makeBase45Fake(
+					from: DigitalCovidCertificate.fake(
+						name: .fake(familyName: "Schneider", givenName: "Saskia", standardizedFamilyName: "SCHNEIDER", standardizedGivenName: "SASKIA"),
+						testEntries: [TestEntry.fake(dateTimeOfSampleCollection: "2021-04-12T16:01:00Z")]
+					),
+					and: CBORWebTokenHeader.fake()
+				)
+				
+				if case let .success(base45) = dcc {
+					registerHealthCertificate(base45: base45)
+				}
+				
+			}
```

Comment:
I don't think we need this, do we?

Reply:
Nope. Totale forgot to code clean-up this. Removed...!